app/src/components/disclosure/styles.tsx

```diff
@@ -0,0 +1,74 @@
+import { css } from "@emotion/react";
+
+export const disclosureGroupCss = css`
+  & > * {
+    width: 100%;
+    .react-aria-Heading {
+      width: 100%;
+      .react-aria-Button[slot="trigger"] {
+        width: 100%;
+      }
+    }
+  }
+`;
+
+export const disclosureCss = css`
+  .react-aria-Heading {
+    margin: 0;
+  }
+
+  .react-aria-Button[slot="trigger"] {
+    // reset trigger styles
+    background: none;
+    border: none;
+    box-shadow: none;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 8px;
+    padding: var(--ac-global-dimension-static-size-100)
+      var(--ac-global-dimension-static-size-100);
+
+    // style trigger
+    color: var(--ac-global-text-color-900);
+    border-bottom: 1px solid var(--ac-global-border-color-default);
+    outline: none;
+    background-color: transparent;
+    &:hover:not([disabled]) {
+      background-color: var(--ac-global-input-field-background-color-active);
+    }
+    &[data-focus-visible] {
+      outline: 1px solid var(--ac-global-input-field-border-color-active);
+      outline-offset: -1px;
+    }
+    &:not([disabled]) {
+      transition: all 0.2s ease-in-out;
+    }
+    &[disabled] {
+      cursor: default;
+      opacity: 0.6;
+    }
+
+    // style trigger icon
+    > svg,
+    > i {
+      rotate: 0deg;
+      transition: rotate 200ms;
+      width: 1em;
+      height: 1em;
+      fill: currentColor;
+    }
+
+    &[data-arrow-position="start"] {
+      flex-direction: row-reverse;
+    }
+  }
+
+  &[data-expanded] .react-aria-Button[slot="trigger"] svg,
+  &[data-expanded] .react-aria-Button[slot="trigger"] i {
+    rotate: 45deg;
```

Comment:
I think the > icon makes more sense when it's placed as a prefix to the title rather than on the end. When it's placed on the right I think it makes more sense to look like GitHub
<img width="941" alt="Screenshot 2025-01-02 at 7 37 05 PM" src="https://github.com/user-attachments/assets/5893c46c-12c2-4370-ae94-ae6b4976000b" />

Reply:
Done